src/components/Settings/index.tsx

```diff
@@ -113,6 +133,21 @@ export default function SettingsTab() {
         hasArrow
         mobileCustomStyle={{ paddingBottom: '40px' }}
       >
+        {location.pathname.indexOf(AppPaths.SWAP) === 0 && (
```

Comment:
How about `location.pathname.startsWith(AppPaths.SWAP) === 0`? I think it's more readable. And maybe this condition should be put in a variable such as `shouldShowGuide` or `isSwapPage`

Reply:
:+1:, but you have mistake maybe. It should be `location.pathname.startsWith(AppPaths.SWAP)` without `=== 0`

Reply:
Exactly :joy:

Reply:
thanks guys, I  updated